src/screens/About/About.tsx

```diff
@@ -44,14 +60,25 @@ export const sidebarItems: TocItem[] = [
         label: aboutContent.futureProjectsHeader,
       },
       {
-        to: '/about#join-us',
-        label: aboutContent.joinUsHeader,
+        to: '/about#contact-us',
+        label: aboutContent.contactUsHeader,
       },
     ],
   },
 ];
 
 // TODO(Chelsi): put these ids into the CMS
+const Button: React.FC<{ to?: string; href?: string; cta: string }> = ({
+  to,
+  href,
+  cta,
+}) => {
+  return (
+    <LargeOutlinedButton to={to} href={href} endIcon={<ArrowForwardIcon />}>
```

Comment:
these buttons need 2 tracking props--`trackingCategory={EventCategory.ABOUT}`  and `trackingLabel={cta}` should do the trick

Reply:
awesome, thank you